numpy/lib/function_base.py

```diff
@@ -3942,7 +3935,7 @@ def percentile(a,
         8. 'median_unbiased'
         9. 'normal_unbiased'
 
-        The first three methods are discontinuous.  NumPy further defines the
+        The first three methods are discontiuous.  NumPy further defines the
```

Comment:
The original spelling was correct.
```suggestion
        The first three methods are discontinuous.  NumPy further defines the
```